packages/client/components/BottomControlBarReadyButton.tsx

```diff
@@ -0,0 +1,183 @@
+import {keyframes} from '@emotion/core'
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React, {ReactNode, useEffect, useState} from 'react'
+import {useFragment} from 'react-relay'
+import {BottomControlBarReadyButton_meeting$key} from '~/__generated__/BottomControlBarReadyButton_meeting.graphql'
+import useAtmosphere from '../hooks/useAtmosphere'
+import isDemoRoute from '../utils/isDemoRoute'
+
+const shake = keyframes`
+  20%, 40%, 60% {
+    padding: 4px 4px 8px;
+    overflow: hidden;
+    background: #a06bd6;
+    color: white;
+    border-radius: 4px;
+    box-shadow: 0px 9px 12px rgba(68, 66, 88, 0.14), 0px 3px 16px rgba(68, 66, 88, 0.12), 0px 5px 6px rgba(68, 66, 88, 0.2);
+  }
+  0% {
+    transform: rotate(0deg);
+  }
+  20% {
+    transform: rotate(-7deg);
+  }
+  40% {
+    transform: rotate(4deg);
+  }
+  60% {
+    transform: rotate(-7deg);
+  }
+  100% {
+    transform: rotate(0deg);
+  }
+`
+
+const BottomNavReadyButton = styled('div')<{shouldAnimate?: boolean; delay: number}>(
+  ({shouldAnimate, delay}) => {
+    return {
+      display: 'flex',
+      alignItems: 'center',
+      justifyContent: 'center',
+      animation: shouldAnimate ? `1s ease-in-out ${delay}s 3 ${shake}` : undefined
+    }
+  }
+)
+
+interface BottomControlBarReadyButton {
+  meetingRef: BottomControlBarReadyButton_meeting$key
+  children: ReactNode
+}
+
+const BottomControlBarReadyButton = (props: BottomControlBarReadyButton) => {
+  const {meetingRef, children} = props
+  const meeting = useFragment(
+    graphql`
+      fragment BottomControlBarReadyButton_meeting on NewMeeting {
+        localPhase {
+          phaseType
+        }
+        localStage {
+          readyCount
+          isViewerReady
+        }
+        meetingMembers {
+          id
+        }
+        ... on RetrospectiveMeeting {
+          endedAt
+          facilitatorUserId
+          localPhase {
+            ... on ReflectPhase {
+              reflectPrompts {
+                id
+                editorIds
+              }
+            }
+          }
+          reflectionGroups {
+            id
+            reflections {
+              id
+              isEditing
+              isDropping
+              isViewerDragging
+            }
+          }
+          viewerMeetingMember {
+            votesRemaining
+          }
+          votesRemaining
+        }
+      }
+    `,
+    meetingRef
+  )
+  const {
+    endedAt,
+    facilitatorUserId,
+    localPhase,
+    localStage,
+    meetingMembers,
+    reflectionGroups,
+    viewerMeetingMember,
+    votesRemaining
+  } = meeting
+  const [delaySeconds, setDelaySeconds] = useState(2)
+  const [shouldAnimate, setShouldAnimate] = useState(false)
+  const atmosphere = useAtmosphere()
+  const {viewerId} = atmosphere
+
+  useEffect(() => {
+    // only enable for facilitatorUser
+    const isFacilitator = viewerId === facilitatorUserId && !endedAt
```

Comment:
-1 I think it wasn't clear in the ticket, but we should also add it to the "Ready" button while we're at it.

Reply:
Aha, it is mentioned in https://github.com/ParabolInc/parabol/issues/6815: `This will only appear for the facilitator.` However, I have made the code logic here directly inside `return()`, i.e. if is not a Facilitator, just render the normal button directly.